lighthouse-extension/app/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "__MSG_appName__",
-  "version": "1.6.0",
+  "version": "2.0.0.1",
```

Comment:
probably best to ditch this change as it might make releasing somewhat weird (unless we don't start with 2.0.0?)